auth_test.py

```diff
@@ -845,29 +845,75 @@ def test_grant_revoke_cleanup(self):
 
         assert_unauthorized(cathy, "SELECT * FROM ks.cf", "User cathy has no SELECT permission on <table ks.cf> or any of its parents")
 
-    def test_permissions_caching(self):
+    def test_permissions_caching_authenticated_user(self):
         """
-        * Launch a one node cluster, with a 2s permission cache
+        This test is to show that the permissions caching in AuthenticatedUser
+        works correctly and revokes the permissions from a logged in user
```

Comment:
Will it be correct to end the sentence?
```suggestion
        works correctly and revokes the permissions from a logged in user.
```
I find it helpful while reading.